chainregistry.go

```diff
@@ -38,12 +38,14 @@ import (
 const (
 	// defaultBitcoinMinHTLCMSat is the default smallest value htlc this
 	// node will accept. This value is proposed in the channel open sequence
-	// and cannot be changed during the life of the channel.
+	// and cannot be changed during the life of the channel. It is zero by
+	// default to allow maximum flexibility in deciding what size payments
+	// to forward.
 	//
 	// All forwarded payments are subjected to the min htlc constraint of
 	// the routing policy of the outgoing channel. This implicitly controls
 	// the minimum htlc value on the incoming channel too.
-	defaultBitcoinMinHTLCInMSat = lnwire.MilliSatoshi(1000)
+	defaultBitcoinMinHTLCInMSat = lnwire.MilliSatoshi(0)
```

Comment:
Why would accepting a zero amount HTLC ever be useful? I'm not even sure we have any tests to exercise if it'll even flow as normal for a point-to-point (direct, no hops) payment.

Reply:
The reason for this is pretty far-fetched. It would prepare for a future where 0 sat payments are possible. It seemed safe to me, as CL is also setting this value to zero. But the direct payment case I didn't consider. Changed to 1 msat.